.github/workflows/snyk-test.yaml

```diff
@@ -27,6 +32,7 @@ jobs:
           $PYTHON -m pip freeze > requirements.txt
           snyk test \
             -d \
+            --fail-on=all \
```

Comment:
This should be `--fail-on=patchable` right? If you want to make sure that test passes for unfixable vulnerabilities
<img width="781" alt="image" src="https://github.com/h2oai/cloud-discovery-py/assets/67860730/4393fe2d-a97a-4220-b5eb-8c578eda7a52">

https://docs.snyk.io/snyk-cli/scan-and-maintain-projects-using-the-cli/advanced-failing-of-builds-in-snyk-cli

Reply:
I believe it's `--fail-on=all`

![image](https://github.com/h2oai/cloud-discovery-py/assets/10218581/92628e0c-da22-40b3-bf40-126aec0bbbce)

Documentation is not super clear but I read it as `all = (patchable + upgradable)`

I want to make it pass only if there's not known fix.

WDYT?

Reply:
Yeah you are correct`all = (patchable + upgradable)`